ontology/yaml/resources/HVAC/entity_types/AHU.yaml

```diff
@@ -5439,3 +5439,106 @@ AHU_BSPC_DX4SC_ECON_EFSS_EFVSC_SFSS_SFVSC_SSPC_RHHC_NONCANONICAL_1:
   - heating_water_valve_percentage_command
   - min_zone_air_relative_humidity_sensor
   - min_zone_air_relative_humidity_setpoint
+
+# TW-NTC-TPKE
+AHU_DSPM_DFVSC_DTM_RDC_OAMC_OA_CHWRC_RCO2M:
+  description: "AHU with discharge air/fan, chilled water valve operation, humidity monitoring with dual dampers monitoring and control ."
+  is_canonical: true
+  implements:
+  - AHU
+  - DSPM
+  - DFVSC
+  - DTM
+  - RDC
+  - OAMC
+  - OA
+  - CHWRC
+  - RCO2M
+  opt_uses:
+    - filter_alarm
+    - low_differential_pressure_alarm
```

Comment:
This is too vague - would expect it to be related to the discharge air

Reply:
I have changed it to low_discharge_static_pressure_alarm